### DISCUSSION TITLE
Client "Group" Tab contains all not assigned devices - option to disable this

### DISCUSSION THREAD

**Discussion by ConGp20** [rustdesk/rustdesk#7290] [Feature Request]
If you create a user and assigned to a group, the rustdesk clients "Group" Tab shows all Devices that are connect to the server and are not assigned to a group or user. Theres no way to disable, that alle not assigned devices are shown.

This is a problem, if you want to give access to other people that should not see these devices.

**Comment by rustdesk** [comment]
So you do not want the user access the unsigned device? then who can access these unassigned device?

**Reply by ConGp20** [reply]
I´d like to have an option to only give access to devices in the same group. So that you can opt out that unassigned devices show up at a specific user

**Reply by grummbeer** [reply]
Users could have a flag e.g. "super user". The super users can manage the groups and group members who see only the devices for there daily work e.g. a specific company unit could be a group.

**Reply by rustdesk** [reply]
We will evaluate this, and listen to more voices before taking action.

**Reply by ConGp20** [reply]
Another Idea would be, that you could assign a default group to all devies, so that they are no longer unassagined. I know, you did the concept the other way round, but maybe this would be an "easy fix"
